test/image_test.py

```diff
@@ -449,6 +472,7 @@ def test_image_run_function(builder_version, servicer, client):
 def test_image_run_function_interactivity(builder_version, servicer, client):
     app = App()
     app.image = Image.debian_slim().pip_install("pandas").run_function(run_f)
```

Comment:
Maybe doesn't matter but still *doing* the assignment here and below.

Reply:
Yeah but we still want to support this syntax. I deliberately made sure some tests are using `app.image`, some just using `image`, and some `App(image=...)` just so we cover all options